### PR TITLE
Combined methods

### PR DESCRIPTION
**Pull request by Jesse193**
This adds the helper method team_accuracy and the methods most_accurate_team and lest_accurate_team as well as tests

### CODE REVIEW

**Review by GlowMunch — APPROVED** [review]
pulled and tested. all pass

File: lib/season.rb [hunk 1]
```diff
@@ -37,4 +37,33 @@ def most_tackles(season)
...
+    create_season(season).each do |row|
+      shots = team_shots_total[row[:team_id]] += row[:shots].to_i
+      goals = team_goals_total[row[:team_id]] += row[:goals].to_i
+      team_accuracy = goals / shots.to_f
+      team_id = row[:team_id]
+      @team_accuracies[team_id] = team_accuracy
+    end
+  end
+  
+  def most_accurate_team(season)
```
Comment (stackmm) [inline lib/season.rb:53]: I tested both methods with different data sets and it returned the correct team every time

File: spec/season_spec.rb [hunk 2]
```diff
@@ -43,4 +43,12 @@
     end
   end
 
+
+  describe "team accuracy" do
+    it "returns most accurate team" do
+      expect(@season.team_accuracy(20122013))
```
Comment (stackmm) [inline spec/season_spec.rb:49]: you could remove line 49 as it doesn't actually test anything (we can do this in a future refactor of the whole test file when all the other methods are added)

**Review by stackmm — APPROVED** [review]
works well

**Pull request merged**